insights/parsers/ls_rsyslog_errorfile.py

```diff
@@ -0,0 +1,38 @@
+"""
+LsRsyslogErrorfile - command ``ls -ln <path>``
+==============================================
+
+The ``ls -ln <path>`` command provides information for the listing of the
+specified directory.
+
+Attributes:
+    data (dict): Dictionary of keys with values in dict.
```

Comment:
I believe this should be `entries`, And please move these `attributes` .... and `examples` into the Parsers docstring instead of the module's docstring.